content/en/docs/FAQ/performance.md

```diff
@@ -3,6 +3,67 @@ title: "Performance and Scalability"
 description: "Questions about Kiali Performance measurements and improvements."
 ---
 
+### What are some Tips for working with a large mesh?
+
+It can be a observability challenge to work with a large mesh. Here are a few things that can be done to improve the situation.
+
+#### Resources and Connectivity
+
+Before talking about Kiali features, it is important to understand that Kiali's performance is dependent on the performance and responsiveness of your metrics database (typically Prometheus), and your tracing store, for installations using tracing. For Prometheus scalability tips, see [Prometheus Tuning]({{< ref "/docs/Configuration/p8s-jaeger-grafana/prometheus#prometheus-tuning" >}}). See [Tempo Tuning]({{< ref "/docs/Configuration/p8s-jaeger-grafana/tracing/tempo#tempo-tuning" >}}) if using Tempo for your trace store.
+
+Only when query performance for metrics and traces is good, can Kiali respond in a reasonable way. So, it is also important to provide sufficient connectivity for the API calls to return information in a timely way.
+
+#### Manual Refresh
+
+By default, Kiali will immediately attempt to populate the page, the default being the Overview page. After the initial page is rendered, most pages will automatically refresh, based on the setting in the "Refresh Interval" dropdown. The default is every 60s. For a large mesh, even the initial page load can be slow, and it can be frustrating if you have to wait for the page to render before being able to enter desired options and/or filters, and then ask for another refresh.
+
+In the dropdown, Kiali offers the "Manual" refresh setting. If selected, Kiali will not refresh the page on a timer. Kiali also offers a "Pause" setting. "Pause" also prevents a timed refresh, but it will refresh on an option or filter change. "Manual" will only refresh on a manual click the refresh button. To ensure that even the initial page load is avoided, the default can be set in the kiali CR:`spec.kiali_feature_flags.ui_defaults.refresh_interval: manual`. With this setting it is possible to "batch" settings changes. For example, when working with the graph you could choose namespaces, update Display settings, and also change Traffic settings, all before rendering the graph.
```

Comment:
```suggestion
In the dropdown, Kiali offers the "Manual" refresh setting. If selected, Kiali will not refresh the page on a timer. Kiali also offers a "Pause" setting. "Pause" also prevents a timed refresh, but it will refresh on an option or filter change. "Manual" will only refresh on a manual click of the refresh button. To ensure that even the initial page load is avoided, the default can be set in the Kiali CR:`spec.kiali_feature_flags.ui_defaults.refresh_interval: manual`. With this setting it is possible to "batch" settings changes. For example, when working with the graph you could choose namespaces, update Display settings, and also change Traffic settings, all before rendering the graph.
```